persistence-modules/spring-boot-persistence-4/src/main/java/com/baeldung/paging/StudentCustomQueryRepository.java

```diff
@@ -0,0 +1,12 @@
+package com.baeldung.paging;
+
+import org.springframework.data.domain.*;
+import org.springframework.data.jpa.repository.*;
+
+public interface StudentCustomQueryRepository extends JpaRepository<Student, String> {
+
+    @Query(value = "SELECT stu FROM Student stu LEFT JOIN FETCH stu.school ",
```

Comment:
can we store these on a file as CONSTANTS

Reply:
Would you like us to do this?

I have searched our current codebase. This is aligned with our current practice in defining custom queries that put the query string with the attribute value.